functional/cypress-tests/cypress/support/widget.js

```diff
@@ -0,0 +1,79 @@
+import { fillDynamicTab } from './widgetDynamicTab'
+
+class Widget {
```

Comment:
Encapsulates widget configuration, and widget related assertions.

Reply:
maybe it is worth mention about it in readme.

Reply:
Yes that is true and i will, but after i finish refactoring - this is just a part of refactor not the final structure.
I want to unblock all hanging PR for now.